### DISCUSSION TITLE
Set inital directory

### DISCUSSION THREAD

**Discussion by schultzter** [nvim-tree/nvim-tree.lua#1121] [Q&A] [answered]
Is it possible to set the intial directory? 

Regardless of how I launch Nvim-qt the directory ends up being something awkward, I would prefer it start in my home directory or even more prefereably in the Documents subdirectory (I'm on Windows so `C:\User\me\Documents` or `~\Documents`).

Thanks

**Comment by kyazdani42** [comment]
I think this is something that should be managed by a rooter plugin (like project.nvim or such). Or maybe hacking an automation on start. nvim-tree does not manage your base directory (depends on how you open it though).

**Comment by schultzter** [comment]
I got it to do what I wanted with following setup:
``` lua
require('nvim-tree').setup {
  open_on_setup = true,
  update_focused_file = {
    enable = true,
    update_cwd = true,
    ignore_list = {},
  }
}
```

**Reply by kyazdani42** [reply]
ah then i've misunderstood what you've asked :)

**Reply by schultzter** [reply]
More likely I didn't know what I was asking when I started this thread! 😄 

Thanks

**Comment by alex-courtis** [comment, marked as answer]
Looks like we have an answer ;)
